modules/templates/helper.go

```diff
@@ -159,6 +159,9 @@ func NewFuncMap() []template.FuncMap {
 			return setting.DisableGitHooks
 		},
 		"TrN": TrN,
+		"UnescapeBS": func(str string) string {
```

Comment:
"BS"? I agree, but maybe we can name it better? 😂

Reply:
BackSlash :) any suggestions for better name? :)

Reply:
UnescapeLocale? It also took me a few minutes to figure out what BS stood for

Reply:
I thought it actually stood for bullshit 😂 I agree with @ethantkoenig that it should be `UnescapeLocale` 🙂

Reply:
It can stand for that also :) It can be also temporary fix if unknwon fixes that in go-ini library to support such escape as it is per ini file format. That is why I did not want it to be so intrusive